FabricExample/ios/_xcode.env

```diff
@@ -0,0 +1,11 @@
+# This `.xcode.env` file is versioned and is used to source the environment
```

Comment:
That's not `.xcode.env`, it's `_xcode.env`

Reply:
I did not create it manually 🤷‍♂️

Reply:
But the other one exists and is basically the same thing